<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>RouteForm</class>
  <widget class="QDialog" name="RouteForm">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>589</width>
        <height>498</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Route</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <widget class="QPushButton" name="pushButtonUp">
      <property name="geometry">
        <rect>
          <x>520</x>
          <y>190</y>
          <width>50</width>
          <height>28</height>
        </rect>
      </property>
      <property name="text">
        <string>up</string>
      </property>
    </widget>
    <widget class="Q3ListBox" name="listBoxRouteWayPoints">
      <property name="geometry">
        <rect>
          <x>300</x>
          <y>70</y>
          <width>210</width>
          <height>350</height>
        </rect>
      </property>
    </widget>
    <widget class="Q3ListBox" name="listBoxDBWayPoints">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>70</y>
          <width>210</width>
          <height>350</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="pushButtonToRight">
      <property name="geometry">
        <rect>
          <x>250</x>
          <y>230</y>
          <width>40</width>
          <height>28</height>
        </rect>
      </property>
      <property name="text">
        <string>&gt;</string>
      </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDown">
      <property name="geometry">
        <rect>
          <x>520</x>
          <y>270</y>
          <width>50</width>
          <height>28</height>
        </rect>
      </property>
      <property name="text">
        <string>down</string>
      </property>
    </widget>
    <widget class="QLabel" name="textLabelName">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>30</y>
          <width>67</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text">
        <string>Name</string>
      </property>
      <property name="wordWrap">
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
      <property name="geometry">
        <rect>
          <x>520</x>
          <y>230</y>
          <width>50</width>
          <height>28</height>
        </rect>
      </property>
      <property name="text">
        <string>X</string>
      </property>
    </widget>
    <widget class="QWidget" name="Layout1">
      <property name="geometry">
        <rect>
          <x>100</x>
          <y>450</y>
          <width>476</width>
          <height>33</height>
        </rect>
      </property>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <spacer name="Horizontal Spacing2">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>20</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QPushButton" name="buttonOk">
            <property name="text">
              <string>&amp;OK</string>
            </property>
            <property name="shortcut">
              <string/>
            </property>
            <property name="autoDefault">
              <bool>true</bool>
            </property>
            <property name="default">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonCancel">
            <property name="text">
              <string>&amp;Cancel</string>
            </property>
            <property name="shortcut">
              <string/>
            </property>
            <property name="autoDefault">
              <bool>true</bool>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
      <property name="geometry">
        <rect>
          <x>80</x>
          <y>30</y>
          <width>160</width>
          <height>19</height>
        </rect>
      </property>
      <property name="maxLength">
        <number>16</number>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>RouteForm</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>RouteForm</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>pushButtonToRight</sender>
      <signal>clicked()</signal>
      <receiver>RouteForm</receiver>
      <slot>toRight()</slot>
    </connection>
    <connection>
      <sender>pushButtonDown</sender>
      <signal>clicked()</signal>
      <receiver>RouteForm</receiver>
      <slot>down()</slot>
    </connection>
    <connection>
      <sender>pushButtonUp</sender>
      <signal>clicked()</signal>
      <receiver>RouteForm</receiver>
      <slot>up()</slot>
    </connection>
    <connection>
      <sender>pushButtonRemove</sender>
      <signal>clicked()</signal>
      <receiver>RouteForm</receiver>
      <slot>remove()</slot>
    </connection>
  </connections>
</ui>
